app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def new
+    @post = Post.new
+  end
+end
```

Comment:
- [ ] As the previous reviewer suggested, please make sure to write the index and show methods for the posts controller as well. 🙏